frontend/src/components/dashboard/aliases/CustomAddressGenerationModal.module.scss

```diff
@@ -0,0 +1,336 @@
+@import "~@mozilla-protocol/core/protocol/css/includes/lib";
+@import "~@mozilla-protocol/core/protocol/css/includes/forms/lib";
+@import "../../../styles/tokens.scss";
+
+.underlay {
+  position: fixed;
+  background-color: rgba($color-black, 0.4);
+  top: 0;
+  bottom: 0;
+  left: 0;
+  right: 0;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+
+  .dialog-wrapper {
+    background: $color-white;
+    width: calc((#{$content-sm} + #{$content-md}) / 2); // 560px
+    max-width: 90%;
+    border-radius: $border-radius-md;
+    box-shadow: $box-shadow-sm;
+    padding: $spacing-md;
+    outline: none;
+    position: relative;
+
+    .invalid-address-wrapper {
+      position: relative;
+      width: 100%;
+      visibility: hidden;
+      opacity: 0;
+      transition: all 0.6s ease;
+
+      &.active {
+        visibility: visible;
+        opacity: 1;
+      }
+
+      .invalid-address-msg {
+        display: flex;
+        position: absolute;
+        box-shadow: $box-shadow-sm;
+        border-radius: 4px;
```

Comment:
Suggestion (non-blocking): I believe we have a variable for `4px`. `$border-radius-sm`